sources/packages/forms/src/form-definitions/institutionprofilecreation.json

```diff
@@ -4,9 +4,7 @@
   "type": "form",
   "name": "institutionprofilecreation",
   "path": "institutionprofilecreation",
-  "tags": [
-    "common"
-  ],
+  "tags": ["common"],
```

Comment:
mm..looks like a format thing 
![image](https://user-images.githubusercontent.com/77353155/196554376-509b1f35-e219-46e5-908c-31d6ee7ff490.png)

Reply:
Thanks @ann-aot after applying JSON and saving, it goes back to the "unformatted way". I'll keep trying before I bug someone :)